.github/workflows/check-catalyst.yaml

```diff
@@ -75,6 +75,7 @@ jobs:
           RT_BUILD_DIR="$(pwd)/runtime-build" \
           QIR_STDLIB_DIR="$(pwd)/qir-stdlib-build" \
           QIR_STDLIB_INCLUDES_DIR="$(pwd)/qir-stdlib-build/include" \
+          ENABLE_LIGHTNING_KOKKOS=ON \
```

Comment:
I think this part also adds a minute to the critical path

Reply:
Yes. Here, we install the runtime with both backend devices and test the integration tests separately in two different jobs.